### DISCUSSION TITLE
Is the latest Terraform AWS Provider version used by default?

### DISCUSSION THREAD

**Discussion by herrsergio** [DontShaveTheYak/cf2tf#264] [Q&A] [answered]
When I use `cf2tf` I noticed that for a resource, it uses an argument that the Terraform AWS Provider I am using, is not supported.

Can you specify the Terraform AWS Provider version to use?

**Comment by shadycuz** [comment, marked as answer]
Hello @herrsergio 

The ability to choose the version of Terraform as well as the underlying AWS provider would be a nice feature to have, but is not currently supported.

**Reply by shadycuz** [reply]
I have created issue #283 and #282
